src/components/Comments.jsx

```diff
@@ -69,6 +74,26 @@ export default function Comments({ article_id }) {
         }
     };
 
+    const handleDelete = (comment_id) => {
+        deleteComment(comment_id)
+            .then(() => {
+                const updatedComments = comments.filter((comment) => {
+                    return comment.comment_id !== comment_id;
+                });
+                setComments(updatedComments);
+            })
+            .then(() => {
+                const updatedCommentError = { ...commentError };
+                updatedCommentError[comment_id] = false;
+                setCommentError(updatedCommentError);
+            })
+            .catch(() => {
+                const updatedCommentError = { ...commentError };
+                updatedCommentError[comment_id] = true;
+                setCommentError(updatedCommentError);
+            });
```

Comment:
Here you are only removing the comment from the front end, so whenever a user refreshes that comment will still remain in the database and as such will render upon the api request. Make sure to use your api to handle the actual deletion of the comment. You may need to go and add the functionality in the backend but that is fine as it's often how we work in the 'real world' ... decide on functionality to add based on UX, and then follow the whole user story through backend and front end!